portfolio/src/main/webapp/scripts/server-dev.js

```diff
@@ -15,15 +15,38 @@
 /**
  * Fetches the previously entered comments from the server and inserts each
  * comment as a list item of the 'comments' <ul> element.
+ *
+ * <p>
+ *
+ * An option to determine the maximum number of comments is also included
+ * using a Query String parameter created from the num-comments form. When
+ * the page is (re-)loaded, the number of comments displayed is determined
+ * from the selection in the previous session. Otherwise, the most recently
+ * submitted number selection will be used. The number of comments will
+ * also never exceed the number of total comments returned from the datastore.
  */
 function getCommentsThread() {
   fetch('/comment-data')
       .then(response => response.json())
       .then((commentList) => {
         const commentThread = document.getElementById('comments-thread');
-        commentList.forEach((comment) => {
-          commentThread.appendChild(createListElement(comment));
-        });
+        const urlParams = new URLSearchParams(window.location.search);
+
+        // Determine the number of comments to display
+        var numComments = urlParams.get('num-comments');
```

Comment:
Prefer let over var. https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Fixed. I am planning on going back to re-read over the JS and Java style guides tomorrow because it seems I missed a few details on the first pass.

Reply:
Did see anything about let because I was reading the [wrong style guide](https://google.github.io/styleguide/javascriptguide.xml), oops.